sdk/resourcemanager/azure-resourcemanager-containerservice/src/test/java/com/azure/resourcemanager/containerservice/KubernetesClustersTests.java

```diff
@@ -626,4 +627,95 @@ public void testUpdateManagedClusterSkuAndKubernetesSupportPlan() {
         Assertions.assertEquals(ManagedClusterSkuTier.FREE, kubernetesCluster.sku().tier());
         Assertions.assertEquals(KubernetesSupportPlan.KUBERNETES_OFFICIAL, kubernetesCluster.innerModel().supportPlan());
     }
+
+    @Test
+    public void canCreateKubernetesClusterWithDisablePublicNetworkAccess() {
+        String aksName = generateRandomResourceName("aks", 15);
+        String dnsPrefix = generateRandomResourceName("dns", 10);
+        String agentPoolName = generateRandomResourceName("ap0", 10);
+        String agentPoolName1 = generateRandomResourceName("ap1", 10);
+
+        String agentPoolResourceGroupName = generateRandomResourceName("pool", 15);
+
+        // create
+        KubernetesCluster kubernetesCluster =
```

Comment:
simplify your resource in test, you don't need all these configure.

a simple one would do
https://github.com/Azure/azure-sdk-for-java/blob/5b6d47ec83296d5a2e4ac539740eea5f3b008791/sdk/resourcemanager/azure-resourcemanager-containerservice/src/test/java/com/azure/resourcemanager/containerservice/KubernetesClustersTests.java#L365-L379